app/views/hyrax/base/_show_actions.html.erb

```diff
@@ -18,7 +18,7 @@
       <% end %>
     </div>
     <div class="col-sm-8 text-right">
-    <% if presenter.editor? %>
+    <% unless presenter.editor? && ESSI.config.dig(:essi, :site_usage) == 'access_only' %>
```

Comment:
I'd like to see the site_usage check as a helper method.

Reply:
Suggested helper was placed in ApplicationHelper because two completely different controller contexts need it: work show and the user profile dropdown.